### DISCUSSION TITLE
Reading a CSV file with semicolon as delimiter

### DISCUSSION THREAD

**Discussion by lukasleitsch** [spatie/simple-excel#133] [Q&A] [answered]
With the release 3.0 the [`useDelimiter()` was removed](https://github.com/spatie/simple-excel/blob/main/UPGRADE.md#removed-usedelimiter-on-simpleexcelwriter). The `SimpleExcelWriter` class allows defining an alternative delimiter.

It seems that the `SimpleExcelReader` don't offer a way to use an alternative delimiter.

Is there a way to read a CSV file with semicolon as delimiter which I am overlooking?

**Comment by klaasgeldof** [comment, marked as answer]
The method is still there. I also didn't find it in the docs, but I just used it and it works as expected.

**Reply by lukasleitsch** [reply]
Yes, you are right. `->useDelelimiter` is still available on `SimpleExcelReader`.

```php
SimpleExcelReader::create(storage_path($file))
            ->useDelimiter(';')
```

**Comment by elburro1887** [comment]
Just stumbled upon this, the documentation README.md is out of date and should to be updated to the newest v3.
